wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Create/ConversationCreationController.swift

```diff
@@ -113,28 +115,45 @@ final class ConversationCreationController: UIViewController {
 
     private lazy var encryptionProtocolSection = {
         let section = ConversationEncryptionProtocolSectionController(values: values)
-        section.isHidden = true
+
         section.tapAction = { sender in
             self.presentEncryptionProtocolPicker(sender: sender) { [weak self] encryptionProtocol in
-                self?.values.encryptionProtocol = encryptionProtocol
-                self?.updateOptions()
+                guard let self else { return }
+
+                self.values.encryptionProtocol = encryptionProtocol
+                self.updateOptions()
+
+                self.reloadOptionsSections()
             }
         }
         return section
     }()
 
+    private func reloadOptionsSections() {
+        guard let collectionView = collectionViewController.collectionView else { return }
+        updateSections()
+
+        let excludedSectionIndex = collectionViewController.sections.startIndex
+        let endIndex = collectionView.numberOfSections
+        let sectionsToReload = IndexSet(integersIn: (excludedSectionIndex + 1)..<endIndex)
```

Comment:
Another way to write this could be:

```
let allIndices = IndexSet(integersIn: 0..<collectionView.numberOfSections)
let indicesToReload = allIndices.dropFirst()
```

Reply:
just to fully understand: what section we avoid to reload? @KaterinaWire

Reply:
@netbe  **Conversation name.** 
Use case:
- I enter a conversation name
- I change the protocol
As a result, I want to keep the conversation name and not reload it. So it will be available **only** for testing, since the protocol change will be hidden from users. We can just ignore it and reload all sections, anyway it's just to improve the user experience of testing.